awxkit/awxkit/api/pages/page.py

```diff
@@ -516,6 +517,26 @@ def get_options(self, page):
 
         return self.options.setdefault(url, options)
 
+    def get_post_fields(self, page):
+        url = page.endpoint if isinstance(page, Page) else str(page)
+        key = get_registered_page(url)
+        if key in self.post_fields:
+            return self.post_fields[key]
+
+        options_page = self.get_options(page)
```

Comment:
You left out the check on `options_page` that was in the original! I know I already approved this, but can we fix this before merging? See L38 in the original. I'm concerned about possibly removing a null check in this updated code. I'm guessing it wasn't covered in the unit tests, either. Don't worry about that, though. I'll follow-up. Just add the check back in here if it still makes sense. If it doesn't make sense here anymore, let me know why.